includes/rest-api/class-sensei-rest-api-home-controller.php

```diff
@@ -0,0 +1,284 @@
+<?php
+/**
+ * Sensei Home REST API.
+ *
+ * @package Sensei\Admin
+ * @since   $$next-version$$
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+/**
+ * Sensei Home REST API endpoints.
+ *
+ * @since $$next-version$$
+ */
+class Sensei_REST_API_Home_Controller extends \WP_REST_Controller {
+
+	/**
+	 * Routes namespace.
+	 *
+	 * @var string
+	 */
+	protected $namespace;
+
+	/**
+	 * Routes prefix.
+	 *
+	 * @var string
+	 */
+	protected $rest_base = 'home';
+
+
+	/**
+	 * Sensei_REST_API_Home_Controller constructor.
+	 *
+	 * @param string $namespace Routes namespace.
+	 */
+	public function __construct( $namespace ) {
+		$this->namespace = $namespace;
+	}
+
+	/**
+	 * Register the REST API endpoints for Home.
+	 */
+	public function register_routes() {
+		$this->register_get_data_route();
+	}
+
+	/**
+	 * Check user permission for REST API access.
+	 *
+	 * @return bool Whether the user can access the Sensei Home REST API.
+	 */
+	public function can_user_access_rest_api() {
+		return current_user_can( 'manage_sensei' );
+	}
+
+	/**
+	 * Register GET / endpoint.
+	 */
+	public function register_get_data_route() {
```

Comment:
Given that this is only one endpoint, do we need this additional method?

Reply:
I will remove if finally we have only a single endpoint :)